lib/telegraf/rack.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'rack'
+
+module Telegraf
+  # Telegraf::Rack
+  #
+  # This rack middleware collects request metrics and sends them to the telegraf
+  # agent. A `Point` data structure is added to the Rack environment to assign
+  # custom tags and values. This point can be accessed using the environment key
+  # in `::Telegraf::Rack::FIELD_NAME`.
```

Comment:
I found the final explanation confusing (although the example resolved it). How about something like "This point can be accessed using the environment key referenced/defined in `::Telegraf::Rack::FIELD_NAME`"?

Reply:
What would you both think about encapsulating this in a method such as `Telegraf::Rack.point_from_rack(env)` (and adapting the documentation accordingly)? IMO, it doesn't hurt hiding this implementation detail, and (with a better name) it could make the whole thing more readable maybe?

Reply:
I'd like to not introduce a method at this point.